blocks/columns/columns.css

```diff
@@ -161,6 +161,13 @@
   max-width: 1200px;
 }
 
+.columns.heading-divider h2,
```

Comment:
@shrotia These changes are taken care as part of PR 
https://github.com/hlxsites/sunstar/pull/351

Please see this page for authoring
https://345-column-block-enhancement--sunstar--hlxsites.hlx.page/sustainability/environment/climate-resource-waste

Reply:
oops, I missed this comment before merging. Let me reconcile